api/cosmos/group/v1/state.cosmos_orm.go

```diff
@@ -0,0 +1,769 @@
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type GroupInfoTable interface {
+	Insert(ctx context.Context, groupInfo *GroupInfo) error
+	InsertReturningID(ctx context.Context, groupInfo *GroupInfo) (uint64, error)
+	Update(ctx context.Context, groupInfo *GroupInfo) error
+	Save(ctx context.Context, groupInfo *GroupInfo) error
+	Delete(ctx context.Context, groupInfo *GroupInfo) error
+	Has(ctx context.Context, id uint64) (found bool, err error)
+	// Get returns nil and an error which responds true to ormerrors.IsNotFound() if the record was not found.
+	Get(ctx context.Context, id uint64) (*GroupInfo, error)
+	List(ctx context.Context, prefixKey GroupInfoIndexKey, opts ...ormlist.Option) (GroupInfoIterator, error)
+	ListRange(ctx context.Context, from, to GroupInfoIndexKey, opts ...ormlist.Option) (GroupInfoIterator, error)
+	DeleteBy(ctx context.Context, prefixKey GroupInfoIndexKey) error
+	DeleteRange(ctx context.Context, from, to GroupInfoIndexKey) error
+
+	doNotImplement()
+}
+
+type GroupInfoIterator struct {
+	ormtable.Iterator
+}
+
+func (i GroupInfoIterator) Value() (*GroupInfo, error) {
+	var groupInfo GroupInfo
+	err := i.UnmarshalMessage(&groupInfo)
+	return &groupInfo, err
+}
+
+type GroupInfoIndexKey interface {
+	id() uint32
+	values() []interface{}
+	groupInfoIndexKey()
+}
+
+// primary key starting index..
+type GroupInfoPrimaryKey = GroupInfoIdIndexKey
+
+type GroupInfoIdIndexKey struct {
+	vs []interface{}
+}
+
+func (x GroupInfoIdIndexKey) id() uint32            { return 0 }
+func (x GroupInfoIdIndexKey) values() []interface{} { return x.vs }
+func (x GroupInfoIdIndexKey) groupInfoIndexKey()    {}
+
+func (this GroupInfoIdIndexKey) WithId(id uint64) GroupInfoIdIndexKey {
+	this.vs = []interface{}{id}
+	return this
+}
+
+type GroupInfoAdminIndexKey struct {
+	vs []interface{}
+}
+
+func (x GroupInfoAdminIndexKey) id() uint32            { return 1 }
+func (x GroupInfoAdminIndexKey) values() []interface{} { return x.vs }
+func (x GroupInfoAdminIndexKey) groupInfoIndexKey()    {}
+
+func (this GroupInfoAdminIndexKey) WithAdmin(admin string) GroupInfoAdminIndexKey {
+	this.vs = []interface{}{admin}
+	return this
+}
+
+type groupInfoTable struct {
+	table ormtable.AutoIncrementTable
+}
+
+func (this groupInfoTable) Insert(ctx context.Context, groupInfo *GroupInfo) error {
+	return this.table.Insert(ctx, groupInfo)
+}
+
+func (this groupInfoTable) Update(ctx context.Context, groupInfo *GroupInfo) error {
+	return this.table.Update(ctx, groupInfo)
+}
+
+func (this groupInfoTable) Save(ctx context.Context, groupInfo *GroupInfo) error {
+	return this.table.Save(ctx, groupInfo)
+}
+
+func (this groupInfoTable) Delete(ctx context.Context, groupInfo *GroupInfo) error {
+	return this.table.Delete(ctx, groupInfo)
+}
+
+func (this groupInfoTable) InsertReturningID(ctx context.Context, groupInfo *GroupInfo) (uint64, error) {
+	return this.table.InsertReturningID(ctx, groupInfo)
```

Comment:
Something is wrong in codegen given it should be `InsertReturningPKey` unless I've done something wrong..